### PR TITLE
Fix Helm integration instructions.

### PR DESCRIPTION
**Pull request by juliandunn**
I did some experimentation using Docker for Mac Edge (bleeding edge) which includes a Kubernetes cluster and added those instructions, plus I found a few problems/improvements:

* For some reason a direct `helm install` of the generated chart
  fails to also install the habitat-operator/habitat-operator into
  K8S. I had to manually install it first. Maybe this is a bug...
* We can export the .hart off the depot directly into a Helm chart,
  thus saving the user from having to clone core-plans and compile
  anything

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by tashimi** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/345341698).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/345341698).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

File: www/source/blog/2018-02-21-Habitat-Helm.html.md [hunk 1]
```diff
@@ -64,15 +59,16 @@ Here we are going to set up Helm in your Kubernetes cluster, and then deploy our
...
-* `helm dependency update /Users/tdrew/core-plans/nginx/results/nginx-1.11.10-20180220023948/` (or whatever your pwd was -- if it's exactly this we need to talk.) Reads your new nginx-<version>-<timestamp>/requirements.yaml file and sees the dependency on the habitat-operator. Pulls down the chart that describes the Habitat Operator and embeds it in your application’s chart
-* `helm install /Users/tdrew/core-plans/nginx/results/nginx-1.11.10-20180220023948/` Has Helm install your Habitat Helm package on your Minikube cluster
+* `helm repo add habitat-operator https://kinvolk.github.io/habitat-operator/helm/charts/stable/` Adds the Habitat Operator Helm repository to your Helm configuration
+* `helm install habitat-operator/habitat-operator` Installs the Habitat Operator into your Kubernetes cluster using its Helm Chart.
+* `helm dependency update nginx-latest` Reads your new nginx-latest/requirements.yaml file and sees the dependency on the habitat-operator. Pulls down the chart that describes the Habitat Operator and embeds it in your application’s chart
```
Comment (zeenix) [inline www/source/blog/2018-02-21-Habitat-Helm.html.md]: If you install operator manually (the previous step you added), this step is completely redundant. You should not need the previous step at all actually.

**Comment by zeenix** [comment]
@julian7 Hi,

> For some reason a direct helm install of the generated chart
> fails to also install the habitat-operator/habitat-operator into
> K8S. I had to manually install it first. Maybe this is a bug...

Do you happen to remember what error you got?

If the chart is not coming from the core repository, the URL has to be manually added first. It should not be needed as `helm` has all the info needed but for some reason it is.

**Comment by julian7** [comment]
@zeenix, you must meant @juliandunn

I haven’t played with helm yet.

**Comment by juliandunn** [comment]
@zeenix I don't remember the error; I just tried to reproduce today from a clean setup and couldn't. It was something like "can't find dependency habitat-operator". Perhaps it was a transient Helm bug.

**Comment by zeenix** [comment]
@julian7 ah yes, so sorry. github should do a better job with suggestions. :)

@juliandunn Thanks for trying to reproduce. Then I'd guess that's either the rare bug in helm or you forgot to add the repository first. I'll submit a new PR to remove the redundant steps and CC you. If you could kindly review it, that would be awesome.

**Comment by zeenix** [comment]
@juliandunn Here it is: https://github.com/habitat-sh/habitat/pull/4690

**Pull request merged**